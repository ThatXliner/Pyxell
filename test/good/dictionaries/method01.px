
print {2: 5, 3: 4}.all(lambda t: t.a<t.b)
print {"": "ab", "ab": "c"}.any(_.b.length==0)
print {'A': true, 'B': false}.filter(_.b)
print {"111": 20.0}.map(lambda t: (t.a.length, "{t.b}{t.b}"))
print {x//2: x for x in 0...7}.fold(lambda t, c: t.a + t.b + c, 0.9)
