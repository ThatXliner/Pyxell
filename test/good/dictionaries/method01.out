true
false
{'A': true}
{3: "2020"}
21.9
